keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/PromotionStatus.kt

```diff
@@ -1,5 +1,32 @@
 package com.netflix.spinnaker.keel.core.api
 
 enum class PromotionStatus {
-  PENDING, APPROVED, DEPLOYING, CURRENT, PREVIOUS, VETOED
+  /**
+   * Waiting on constraint evaluation before being approved
```

Comment:
Thanks for documenting the enum values! ❤️

Reply:
+1!